pkg/config/config.go

```diff
@@ -1654,6 +1655,7 @@ func setupFipsEndpoints(config Config) error {
 		appsecEvents               = 11
 		orchestratorExplorer       = 12
 		runtimeSecurity            = 13
+		networkDevicesNetflow      = 15
```

Comment:
not a blocker and just out of curiosity: why the skip of 14 (could we add a comment about it)?

Reply:
Because 14 is used in this PR https://github.com/DataDog/datadog-agent/pull/20230/files which will be merged soon 🤞